build-tools/packages/build-tools/src/common/fluidBuild.ts

```diff
@@ -0,0 +1,36 @@
+import registerDebug from "debug";
+import { loadFluidBuildConfig } from "./fluidUtils";
+import { findGitRootSync } from "./utils";
+
+/**
+ * The default name of the fluid-build executable. This is used for identifying fluid-build tasks and for trace log
+ * entries.
+ */
+export const defaultExecutableName = "fluid-build";
+const repoRoot = findGitRootSync();
+const config = loadFluidBuildConfig(repoRoot);
+
+/**
+ *
+ */
+const FLUID_BUILD_EXE = config.executableNames?.[0] ?? defaultExecutableName;
```

Comment:
I am wondering whether that should just be detected and configured at the entrypoint?

Reply:
Yes, this can be passed in. Will revisit.